### PR TITLE
Release notes for Portal 2.4.0

### PR DESCRIPTION
**Pull request by ccouzens**

### CODE REVIEW

File: articles/portal/ptl-ref-release-notes.md [hunk 1]
```diff
@@ -19,6 +19,16 @@ toc_mdlink: ptl-ref-release-notes.md
 
 This article provides a list of completed Portal releases.
 
+## Release 2.4.0
+
+Date: 28th October 2020
+
```
Comment (m-digman) [inline articles/portal/ptl-ref-release-notes.md:25]: @ccouzens in line with other updates, can you add "Feature update" please.

Comment (ccouzens) [reply articles/portal/ptl-ref-release-notes.md:25]: Done

**Comment by ccouzens** [comment]
Note that it hasn't been deployed yet, but I'll be surprised if it doesn't happen today

**Review by myoung-ukcloud — APPROVED** [review]
LGTM

**Review by Thoma1999 — APPROVED** [review]

**Review by suehighmoor — APPROVED** [review]
LGTM

**Pull request merged**